backend/satellite_tools/reposync.py

```diff
@@ -684,9 +684,9 @@ def sync(self, update_repodata=True):
         fileutils.createPath(os.path.join(CFG.MOUNT_POINT, 'rhn'))  # if the directory exists update ownership only
         for root, dirs, files in os.walk(os.path.join(CFG.MOUNT_POINT, 'rhn')):
             for d in dirs:
-                fileutils.setPermsPath(os.path.join(root, d), group=APACHE_GROUP)
+                fileutils.setPermsPath(os.path.join(root, d), group=APACHE_GROUP, chmod=int('0644', 8))
```

Comment:
Oh, here we iterate over directories. They need of cause 755 permissions. Below we iterate over files and they need 644.

Reply:
I saw that, thought the same and then just continued...